tests/libtransmission/torrent-files-test.cc

```diff
@@ -143,30 +143,34 @@ TEST_F(TorrentFilesTest, hasAnyLocalData)
 
 TEST_F(TorrentFilesTest, isSubpathPortable)
 {
+#ifdef _WIN32
+#define PORTABLE_UNIX_NOT_WIN32 false
+#else
+#define PORTABLE_UNIX_NOT_WIN32 true
+#endif
```

Comment:
Best practice is to undef it after being used, like right after Tests array.

Reply:
https://github.com/transmission/transmission/pull/6187#discussion_r1378433852